sklearn/model_selection/tests/test_split.py

```diff
@@ -1038,36 +974,21 @@ def test_get_n_splits_for_repeated_stratified_kfold():
     assert expected_n_splits == rskf.get_n_splits()
 
 
-def test_repeated_stratified_kfold_determinstic_split():
```

Comment:
replaced by `test_random_state_reproducibility` and `test_repeated_folds_are_different`